rekuho/src/lib/event.ts

```diff
@@ -38,3 +43,18 @@ export const keyboardToKeyInput = (event: KeyboardEvent): KeyInput | null => {
 
   return key ? { modifiers, key } : null
 }
+
+const wheelDelta = (event: WheelEvent): number => {
+  switch (event.deltaMode) {
+    case WheelEvent.DOM_DELTA_PIXEL:
+      console.log(`wheel_event { delta: ${event.deltaY} }`)
+      break
+    case WheelEvent.DOM_DELTA_LINE:
+      console.error("unhandled page wheel line mode")
+      break
+    case WheelEvent.DOM_DELTA_PAGE:
+      console.error("unhandled page wheel delta mode")
+      break
+  }
+  return event.deltaY
```

Comment:
why do we have this code twice, once here and once in Portion.svelte? this should just be a utility function or sth that's used in the other module, ideally